testing/benchmark/Makefile

```diff
@@ -64,8 +63,19 @@ apply:
 destroy:
 	@terraform destroy -auto-approve
 
+.PHONY: log-benckmark-profile
+log-benckmark-profile: 
+	@echo "Starting benchmarks with $(BENCHMARK_PROFILE) profile..."	
+	@echo "Benchmark warmup events: $$BENCHMARK_WARMUP"
+	@echo "Benchmarks count: $$BENCHMARK_COUNT"
+	@echo "Benchmark duration: $$BENCHMARK_TIME"
+	@echo "Benchmark run comand : $$BENCHMARK_RUN"
+
+# BENCHMARK_RUN, BENCHMARK_TIME, BENCHMARK_COUNT, BENCHMARK_WARMUP, BENCHMARK_AGENTS, BENCHMARK_DETAILED
+#    are defined in benchmark-profiles dir
+# 
 .PHONY: run-benchmark
-run-benchmark:
+run-benchmark: log-benckmark-profile
```

Comment:
I have not seen how some metadata is added to the benchmark report file, sorry if it's somewhere, I wonder whether it's possible to add some attributes to the index such as the git commit, branch name and other metadata.

The apm-agent-nodejs did something like that in https://github.com/elastic/apm-agent-nodejs/pull/1145/files#diff-bd398891a1243b93041e337f1c64c1554877fd03a00e1b4b89d4b41ddbc8faa4R23-R37

This will help to narrow down the search in Kibana, for instance, creating graphs for the main branch versus release branches.

Additionally, all the profilers can be added as metadata too

Reply:
By default, we're indexing the following attributes: https://github.com/elastic/apm-server/blob/931271c62a9024fa4b23a12ec574b42db9301b11/testing/benchmark/Makefile#L78-L86

Different branches would use a different index, for instance, we probably want to keep the history of APM Server's performance in the `main` branch, but we'd delete indices that have a PR suffix after they're stale.

Reply:
No strong opinion about the index strategy, but maybe it might be good, from the user experience, to align with what the agents do with one index and different attributes? What do you think?

Reply:
I think you're right, it would be better to re-use the same index in order to visualize other branch results in the same dashboard. @pazone I think it may be worth to just index data to `apmbench` and add the branch name to the document indexed document properties.

Reply:
Now we use the `apmbench-v2` index. Added branch and profiles tags.